server/client_test.go

```diff
@@ -2444,7 +2443,7 @@ func TestClientClampMaxSubsErrReport(t *testing.T) {
 	s1 := RunServer(o1)
 	defer s1.Shutdown()
 
-	l := &captureErrorLogger{errCh: make(chan string, 10)}
+	l := &captureLogger{logCh: make(chan string, 10)}
```

Comment:
I fail to see why all tests that used captureErrorLogger{} need to be changed. For instance, I did just the client modification in client.go, and run this test (without modification) and it passes. Ditto for the gateway test, etc.. I think only tests that did check for TLS handshake should be changed to go to debug, and one can very well create a dedicated capture logger to check that only a given err/debug/whatever message matches before inserting in the channel (to avoid the test itself to have to deal with err/debug/whatever traces that are not relevant.

Reply:
The only reason most of them changed is because I renamed the struct as its not anymore a *error* logger.

Reply:
I've updated this - didnt rename the struct so I think its incorrectly named now but at least the change set is smaller and added a suggested filter to the one